tools/test.py

```diff
@@ -1479,6 +1500,17 @@ def Main():
   def DoSkip(case):
     return SKIP in case.outcomes or SLOW in case.outcomes
   cases_to_run = [ c for c in all_cases if not DoSkip(c) ]
+  if options.run != None:
```

Comment:
In this case it would be `if not options.run is None:` and that's rather wordy.  Personally, I can live with `!=` here.